### PR TITLE
Add mainnet docs and update testnet docs.

### PR DESCRIPTION
**Pull request by dzmitryhil**

### CODE REVIEW

File: deploy/scripts/init-pass.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+read -r -p "Have you already setup a gpgkey that you would like to use for a pass store?(Y/n)" has_gpg
+if [ "$has_gpg" = "y" ] || [ "$has_gpg" = "Y" ]; then
+	:
+else
+	gpg --gen-key
+	gpg-connect-agent reloadagent /bye # restart gpg agent
+fi
+
+echo "Initializing password store utility"
+pass init onomy
```
Comment (Parth4337) [inline deploy/scripts/init-pass.sh]: We dont need this script to run the full node

File: deploy/scripts/init-testnet-full-node.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ ONOMY_APP_CONFIG="$ONOMY_HOME_CONFIG/app.toml"
 # Chain ID flag
 ONOMY_CHAINID_FLAG="--chain-id $CHAINID"
 # Seeds IPs
-ONOMY_SEEDS_DEFAULT_IPS="64.9.136.119,64.9.136.120,54.88.212.224"
+ONOMY_SEEDS_DEFAULT_IPS="3.88.76.0,3.226.87.93,54.88.212.224"
```
Comment (Parth4337) [inline deploy/scripts/init-testnet-full-node.sh]: I have removed 3rd seed from testnet. Remove the IP `3.226.87.93`

File: deploy/scripts/init-pass.sh (hunk 1 above)
Comment (dzmitryhil) [reply deploy/scripts/init-pass.sh]: Agree, removed.

File: deploy/scripts/init-testnet-full-node.sh (hunk 2 above)
Comment (dzmitryhil) [reply deploy/scripts/init-testnet-full-node.sh]: Updated.

**Review by Parth4337 — APPROVED** [review]

**Pull request merged**